index.js

```diff
@@ -1,56 +1,52 @@
-'use strict';
-var childProcess = require('child_process');
-var neatCsv = require('neat-csv');
-var sec = require('sec');
-var pify = require('pify');
-var Promise = require('pinkie-promise');
-
-module.exports = function (opts) {
+const childProcess = require('child_process');
+const pify = require('pify');
+const neatCsv = require('neat-csv');
+const sec = require('sec');
+
+module.exports = opts => {
 	if (process.platform !== 'win32') {
 		return Promise.reject(new Error('Windows only'));
 	}
 
-	opts = opts || {};
+	const options = opts || {};
 
-	var args = ['/v', '/nh', '/fo', 'CSV'];
+	const args = ['/v', '/nh', '/fo', 'csv'];
 
-	if (opts.system && opts.username && opts.password) {
-		args.push('/s', opts.system, '/u', opts.username, '/p', opts.password);
+	if (options.system && options.username && options.password) {
+		args.push('/s', options.system, '/u', options.username, '/p', options.password);
 	}
 
-	if (Array.isArray(opts.filter) && opts.filter.length) {
-		opts.filter.forEach(function (el) {
-			args.push('/fi', el);
-		});
+	if (Array.isArray(options.filter)) {
+		options.filter.forEach(fi => args.push('/fi', fi));
```

Comment:
It's faster. I'm trying to use it over forEach whenever possible. So for consistency too. And I don't like inline arrow functions for things that doesn't return anything. Makes the code harder to read.